src/core/tests/frontend/paddle/test_models/gen_scripts/generate_roi_align.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# roi_align paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle
+import sys
+
+
+def make_rois(batch_size, width, height, pooled_width, pooled_height, spatial_scale, roi_per_batch):
+    rois = []
+    rois_num = []
+    for bno in range(batch_size):
+        for i in range(roi_per_batch):
+            x1 = np.random.randint(
+                0, width // spatial_scale - pooled_width)
+            y1 = np.random.randint(
+                0, height // spatial_scale - pooled_height)
+
+            x2 = np.random.randint(x1 + pooled_width,
+                                   width // spatial_scale)
+            y2 = np.random.randint(
+                y1 + pooled_height, height // spatial_scale)
+
+            roi = [x1, y1, x2, y2]
+            rois.append(roi)
+        rois_num.append(len(rois))
+    rois = np.array(rois).astype("float32")
+    rois_num = np.array(rois_num).astype("int32")
+
+    return rois, rois_num
+
+
+def roi_align(name: str, x_data, rois_data, rois_num_data, pooled_height, pooled_width, spatial_scale, sampling_ratio):
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        x = paddle.static.data(
+            name='x', shape=x_data.shape, dtype=x_data.dtype)
+        rois = paddle.static.data(
+            name='rois', shape=rois_data.shape, dtype=rois_data.dtype)
+        rois_num = paddle.static.data(
+            name='rois_num', shape=rois_num_data.shape, dtype=rois_num_data.dtype)
+        # TODO: 'aligned' attribute is not supported by Paddle 2.1
```

Comment:
Hi, @ceciliapeng2011 : as we synced offline, 'aligned' attribute is the feature of  ROIAlign_9 which is another story. let's try to close this PR first, because it's blocked other PRs.
about Add "paddledet==2.1.0" to CI, because it's not needed by this PR, let's add it in the following PRs, so that we can fully evaluate whether this "paddledet" new library may bring in problems to our current CI.